cmd/portal/config/config.go

```diff
@@ -0,0 +1,106 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fatih/structs"
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+)
+
+const (
+	CONFIGS_DIR_NAME       = ".config"
+	PORTAL_CONFIG_DIR_NAME = "portal"
+	CONFIG_FILE_NAME       = "config"
+	CONFIG_FILE_EXT        = "yml"
+
+	StyleRich = "rich"
+	StyleRaw  = "raw"
+)
+
+type Config struct {
+	Relay                string `mapstructure:"relay"`
+	Verbose              bool   `mapstructure:"verbose"`
+	PromptOverwriteFiles bool   `mapstructure:"prompt_overwrite_files"`
+	RelayPort            int    `mapstructure:"relay_port"`
+	TuiStyle             string `mapstructure:"tui_style"`
+}
+
+func GetDefault() Config {
+	return Config{
+		Relay:                "167.71.65.96:80",
```

Comment:
While we're at it with many changes in this PR, we could also squeeze in a change of the default relay to simply `spatiumportae.com`